build/codegen.sh

```diff
@@ -22,6 +22,7 @@ set -o xtrace
 
 export GO111MODULE=on
 go mod download
+go get k8s.io/code-generator@$(go list -f '{{.Version}}' -m k8s.io/code-generator)
```

Comment:
We shouldn't add `go mod` and `go get` commands in codegen. These are expected to be run while upgrading K8s deps.

Reply:
we don't have code-generator in go.mod. even if we add it, it gets removed as we are not using it anywhere in code

Reply:
We should check what has changed that is causing this?

Reply:
Somehow, after this PR, we have remove the entry for this package (for 0.26.+ version) from go.sum
https://github.com/kanisterio/kanister/pull/1862/files#diff-3295df7234525439d778f1b282d146a4f1ff6b415248aaac074e8042d9f42d63L1325

Reply:
code-generator entry for new/upgraded version was not added because we are not using it anywhere in code. is it okay if we just import this package in `pkg/apis//cr/v1alpha1/register.go` or `pkg/apis//cr/v1alpha1/doc.go` as it is required only for generating clients?

Reply:
added empty import in `pkg/apis/cr/v1alpha1/doc.go`